src/workerd/api/crypto/impl.c++

```diff
@@ -4,6 +4,8 @@
 
 #include "impl.h"
 
+#include "simdutf.h"
```

Comment:
undo?

Reply:
This is actually required. We include simdutf on value.h without needing it, and removing it requires this particular change.